### PR TITLE
feat(): add profile page

### PR DESCRIPTION
**Pull request by john-patterson**
Solves issue https://github.com/jacobnisnevich/bartenders-manual/issues/19.

Notice when you are logged in, there is now a profile tab. These tables have links to the ingredients and cocktails pages. Note the row styling:
image

Finally, the user's box on both the ingredients and cocktails page has been revamped to have the proper link styling and link to user pages:
image
image

Clicking this will bring you to another user's profile:

image

### CODE REVIEW

File: src/client/components/Cocktail.tsx [hunk 1]
```diff
@@ -119,8 +119,8 @@ export default function Cocktail(props: CocktailProps): JSX.Element {
                   {userIngredients.map(i => i.ingredient.id).includes(cocktailIngredient.ingredient.id) ? (
                     <ImCheckmark className="icon-ingredient-owned" />
                   ) : (
-                    <ImCross className="icon-ingredient-missing" />
-                  )}
+                      <ImCross className="icon-ingredient-missing" />
```
Comment (john-patterson) [inline src/client/components/Cocktail.tsx:122]: Undo this.

File: data/schema.graphql [hunk 2]
```diff
@@ -99,12 +100,12 @@ type Mutation {
   createCocktailSource(data: CreateCocktailSourceInput!): CocktailSource!
   mergeIngredients(primaryIngredientId: String!, secondaryIngredientId: String!): Boolean!
   createIngredient(data: CreateIngredientInput!): Ingredient
+  createUserCocktails(cocktailIds: [String!]!): [UserCocktail!]!
```
Comment (john-patterson) [inline data/schema.graphql:103]: Undo this.

File: src/client/components/Header.tsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ import LoginButton from './LoginButton';
 import LogoutButton from './LogOutButton';
 import Menu from './Menu';
 import ShoppingList from './ShoppingList';
+import { MenuList } from 'react-select/src/components/Menu';
```
Comment (john-patterson) [inline src/client/components/Header.tsx:28]: Revert this.

**Review by jacobnisnevich — APPROVED** [review]
lgtm!

**Comment by AlexNisnevich** [comment]
👍

**Pull request merged**